discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java

```diff
@@ -0,0 +1,72 @@
+package com.nepxion.discovery.common.entity;
+
+import com.nepxion.discovery.common.constant.DiscoveryConstant;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static com.nepxion.discovery.common.entity.InstanceEntityWrapper.*;
+
+public class InstanceEntityWrapperTest {
+
+    private InstanceEntity instanceEntity = new InstanceEntity();
+    private HashMap<String, String> hashMap = new HashMap<>();
+
+    @Before
+    public void setUp() {
+        instanceEntity.setMetadata(hashMap);
+    }
+
+    @Test
+    public void testGetContextPath() {
+        assertNull(getContextPath(instanceEntity));
+    }
+
+    @Test
+    public void testGetGroup() {
+        assertEquals("", getGroup(instanceEntity));
```

Comment:
Would prefer a non-empty group

Reply:
I tested for a non-empty group in line 35.
The method will only return content if the value "spring.application.group.key" exists else it returns ""